test/srt/run_suite.py

```diff
@@ -368,7 +369,7 @@ def auto_partition(files, rank, size):
     arg_parser.add_argument(
         "--timeout-per-file",
         type=int,
-        default=1200,
+        default=1500 if is_hip() else 1200,
```

Comment:
we should not increase this timeout, instead, we should split the files and make each of them run faster.